src/main/config/config.c

```diff
@@ -136,6 +136,8 @@ void resetPidProfile(pidProfile_t *pidProfile)
 
     pidProfile->acc_soft_lpf_hz = 15;
     pidProfile->gyro_soft_lpf_hz = 60;
+	pidProfile->gyro_soft_notch_cutoff_hz = 130;
+	pidProfile->gyro_soft_notch_hz = 260;
```

Comment:
You are right, the majority of Inav craft will be bigger and need lower frequency. I just copied the BF defaults. I could change it to 84 center and 42 cutoff, which I use.